script.js

```diff
@@ -1,4 +1,208 @@
+    }
+  }
+  return newDeck;
 };
+// Get a random index ranging from 0 (inclusive) to max (exclusive).
+const getRandomIndex = (max) => Math.floor(Math.random() * max);
+// Shuffle an array of cards
+const shuffleCards = (deck) => {
+  // Loop over the card deck array once
+  for (let currentIndex = 0; currentIndex < deck.length; currentIndex += 1) {
+    // Select a random index in the deck
+    const randomIndex = getRandomIndex(deck.length);
+    // Select the card that corresponds to randomIndex
+    const randomCard = deck[randomIndex];
+    // Select the card that corresponds to currentIndex
+    const currentCard = deck[currentIndex];
+    // Swap positions of randomCard and currentCard in the deck
+    deck[currentIndex] = randomCard;
+    deck[randomIndex] = currentCard;
+  }
+  // Return the shuffled deck
+  return deck;
+};
+
+//define global variables
+var playerCardPlaceholders = document.getElementsByClassName("p-card");
+var computerCardPlaceholders = document.getElementsByClassName("c-card");
+var deck = makeDeck();
+var shuffledDeck = shuffleCards(deck);
+var playButton = document.getElementById("play-button");
+var hitButton = document.getElementById("hit-button");
+var standButton = document.getElementById("stand-button");
+var restartButton = document.getElementById("restart-button");
+var outputMessageElement = document.getElementById("output");
+
+var playerCards = [];
+var computerCards = [];
+var playerScore = 0;
+var computerScore = 0;
+
+//defind helper functions
+function checkBlackJacket(num1, num2) {
+  if ((num1 == 1 && num2 == 10) || (num1 == 10 && num2 == 1)) {
+    score = 21;
+  } else {
+    score = 0;
+  }
+  return score;
+}
+
+function compareScore(num1, num2) {
+  var winner;
+  if ((num1 == 21 && num2 == 21) || (num1 > 21 && num2 > 21)) {
+    winner = "It's a draw!";
+  } else if (num1 < 21 && num2 < 21) {
+    if (21 - num1 < 21 - num2) {
+      winner = "The winner is Player.";
+    } else if (21 - num1 > 21 - num2) {
+      winner = "The winner is Computer.";
+    } else {
+      winner = "It's a Draw!";
+    }
+  } else if (num1 > 21 && num2 <= 21) {
+    winner = "The winner is Computer.";
+  } else if (num2 > 21 && num1 <= 21) {
+    winner = "The winner is Player.";
+  }
+  return winner;
+}
+
+playButton.addEventListener("click", () => {
+  for (var i = 0; i < 2; i++) {
+    var card = shuffledDeck.pop();
+    playerCards.push(card);
+    playerCardPlaceholders[i].src = card.img;
+  }
+  playerScore = checkBlackJacket(playerCards[0].value, playerCards[1].value);
+
+  if (playerScore == 21) {
+    playerScore = playerScore;
+  } else {
+    playerScore = playerCards[0].value + playerCards[1].value;
+  }
+
+  for (var i = 0; i < 2; i++) {
+    var card = shuffledDeck.pop();
+    computerCards.push(card);
+    computerCardPlaceholders[i].src = card.img;
+  }
+
+  computerScore = checkBlackJacket(
+    computerCards[0].value,
+    computerCards[1].value
+  );
+  if (computerScore == 21) {
```

Comment:
this if else statement also seems to be repeated.. a function would be helpful(: